Call requires API level 33 (current min is 29): android.system.SystemCleaner#cleaner

I am getting this error on build:
```

Lint found 3 errors, 0 warnings. First failure:

/build/generated/source/uniffi/debug/java/com/julian_baumann/data_rct/DataRCT.kt:1108: Error: Call requires API level 33 (current min is 29): android.system.SystemCleaner#cleaner [NewApi]
    val cleaner = android.system.SystemCleaner.cleaner()
                                               ~~~~~~~
```

Even tho the following check exists, gradle lint seems to ignore it.

```kotlin
private fun UniffiCleaner.Companion.create(): UniffiCleaner =
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
        AndroidSystemCleaner()
    } else {
        UniffiJnaCleaner()
    }
```

I am using UniFFI version `0.26.1`

Any ideas on how to resolve this?

--- a/uniffi_bindgen/src/bindings/kotlin/templates/ObjectCleanerHelperAndroid.kt
+++ b/uniffi_bindgen/src/bindings/kotlin/templates/ObjectCleanerHelperAndroid.kt
@@ -17,6 +17,7 @@ private class AndroidSystemCleaner : UniffiCleaner {
         AndroidSystemCleanable(cleaner.register(value, cleanUpTask))
 }
 
+@RequiresApi(Build.VERSION_CODES.TIRAMISU)
 private class AndroidSystemCleanable(
     private val cleanable: java.lang.ref.Cleaner.Cleanable,
 ) : UniffiCleaner.Cleanable {

--- a/uniffi_bindgen/src/bindings/kotlin/templates/ObjectCleanerHelperAndroid.kt
+++ b/uniffi_bindgen/src/bindings/kotlin/templates/ObjectCleanerHelperAndroid.kt
@@ -1,4 +1,5 @@
 {{- self.add_import("android.os.Build") }}
+{{- self.add_import("androidx.annotation.RequiresApi") }}
 
 private fun UniffiCleaner.Companion.create(): UniffiCleaner =
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
@@ -8,6 +9,7 @@ private fun UniffiCleaner.Companion.create(): UniffiCleaner =
     }
 
 // The SystemCleaner, available from API Level 33.
+@RequiresApi(Build.VERSION_CODES.TIRAMISU)
 private class AndroidSystemCleaner : UniffiCleaner {
     val cleaner = android.system.SystemCleaner.cleaner()
 

